python/torch_mlir_e2e_test/test_suite/reduction.py

```diff
@@ -678,6 +678,46 @@ def forward(self, a):
 def ReduceL3NormKeepDimModule_basic(module, tu: TestUtils):
     module.forward(torch.rand(3, 4, 5))
 
+# ==============================================================================
+
+class NormScalarOptDimModule(torch.nn.Module):
+    def __init__(self) -> None:
+        super().__init__()
+        self.p = 3.0
+
+    @export 
+    @annotate_args([
+        None,
+        ([-1, -1, -1], torch.float32, True),
+    ])  
+    def forward(self, a):
+        return torch.ops.aten.norm(a, self.p, dim=[0, 1], keepdim=False)
+
+@register_test_case(module_factory=lambda: NormScalarOptDimModule())
+def NormScalarOptDimModule_basic(module, tu: TestUtils):
+    module.forward(torch.rand(3, 4, 5))
+
+
+# ==============================================================================
+
+class NormScalarOptDimKeepDimModule(torch.nn.Module):
+    def __init__(self) -> None:
+        super().__init__()
+        self.p = 3.0
+
+    @export 
+    @annotate_args([
+        None,
+        ([-1, -1, -1], torch.float32, True),
+    ])  
+    def forward(self, a):
+        return torch.ops.aten.norm(a, self.p, dim=[0, 1], keepdim=True)
+
+@register_test_case(module_factory=lambda: NormScalarOptDimKeepDimModule())
+def NormScalarOptDimKeepDimModule_basic(module, tu: TestUtils):
+    module.forward(torch.rand(3, 4, 5))
```

Comment:
nit: use `tu.rand` to generate the random tensors. (I know there are some tests in this file that do not follow this convention. I'll be fixing this soon). See:

https://github.com/llvm/torch-mlir/blob/main/docs/adding_an_e2e_test.md#test-case-function

Reply:
done